lib/txdb/handlers/hook_handler.rb

```diff
@@ -0,0 +1,57 @@
+require 'txgh'
+require 'uri'
+
+module Txdb
+  module Handlers
+    class HookHandler
+      class << self
+        def handle_request(request)
+          new(request).handle
+        end
+      end
+
+      include ResponseHelpers
+
+      attr_reader :request
+
+      def initialize(request)
+        @request = request
+      end
+
+      def handle
+        locale = payload['language']
+
+        tables.each do |table|
+          content = table.database.transifex_api.download(table.resource, locale)
+          table.write_content(content, locale)
```

Comment:
So, a more general comment: This is an IO method that presumably dispatches work to some external process, so I could imagine the response from such would be a confirmation message, an echo of the data back, `true`, or some other positive condition. I can just as easily imagine error conditions would be somewhat the same (e.g., a status code indicating write failure).

Would it be possible to implement a bang-method version of `write_content!` that ensures the transactions are correctly written to the database and would raise an error if not? This would allow you to bubble up errors to the API (assuming that's what you would want to happen).